content/blog/2021/08/2021-02-08.adoc

```diff
@@ -0,0 +1,167 @@
+---
+:layout: post
+:title: "CloudEvents Plugin for Jenkins: Interoperability between Jenkins and CI/CD Tools"
+:tags:
+- gsoc
+- gsoc2021
+- cloudevents
+- interoperability
+- cloud-native
+authors:
+- ShrutiC-git
+description: >
+  This is an introductory blog to the new CloudEvents plugin for Jenkins, developed as a GSoC'21 project. In this blog, we will be looking at how the integration between Jenkins and CloudEvents makes working across a suite of CI/CD tools easier. CloudEvents is the industry-adopted specification for describing events, and with the release of this plugin, Jenkins is now CloudEvents-compliant.
+opengraph:
+  image:/images/post-images/2021-08-cloudevents-plugin/jcdevents.JPG
+---
+
+image:/images/post-images/2021-08-cloudevents-plugin/jcdevents.JPG[CloudEvents Plugin for Jenkins,  align="center"]
+
+== The What, Why and How of Interoperability
+
+With workloads and teams becoming more diverse and complex, there is an increasing need to automate various tasks in the CI/CD ecosystem of an application as a way to decrease complexity that can come with CI/CD. A more diverse team working across different aspects of the application requires a diverse suite of CI/CD tools too, to test and deliver to a wide range of users. More often than not, we need these tools to work together and exchange data to form an effective CI/CD pipeline. However, chaining multiple services together can very easily increase complexity.
```

Comment:
Avoid long lines. They are harder to review. Prefer short lines by inserting a newline after punctuation marks line commas and periods. Something to remember for future posts.